ack/readers/google_dbm/cli.py

```diff
@@ -33,7 +33,8 @@
 @click.option("--dbm-request-type", type=click.Choice(POSSIBLE_REQUEST_TYPES), required=True)
 @click.option("--dbm-query-id")
 @click.option("--dbm-query-title")
-@click.option("--dbm-query-frequency", default="ONE_TIME")
+@click.option("--dbm-query-frequency", type=click.Choice(POSSIBLE_FREQUENCIES), default="ONE_TIME")
+@click.option("--dbm-query-timezone-code", type=click.Choice(POSSIBLE_TIMEZONE_CODES), default="America/New_York")
```

Comment:
Could you add this new option to the [documentation](https://artefactory.github.io/artefactory-connectors-kit/readers.html#google-doubleclick-bid-manager-reader-dbm) ? A small guide on how to use sphinx is available [here](https://artefactory.github.io/artefactory-connectors-kit/contributing.html#documentation).

Reply:
Yes thank you